samples/system-test/detect.test.js

```diff
@@ -215,6 +241,8 @@ test(`should read a document from a local file`, async t => {
     cwd
   );
   t.true(output.includes('Google Cloud Platform'));
+  t.true(output.includes('Word text: Cloud'));
+  t.true(output.includes('Word confidence: 0.9'));
```

Comment:
How consistent is the `0.9` number, will that not change over time?

Reply:
Good question. If it's not 0.9xxx confident about *at least one of the words*, then there's a problem with the API. It's very consistent. Intentionally only went to the first .9 decimal place (often confidence may be .99xxx or .98xxx)